src/utils/IdealGraphVisualizer/ServerCompiler/src/main/java/com/sun/hotspot/igv/servercompiler/ServerCompilerScheduler.java

```diff
@@ -781,10 +798,13 @@ public void markCFGNodes() {
                 n.isCFG = true;
             } else if (n.inputNode.getProperties().get("type").equals("bottom")
                        && n.preds.size() > 0 &&
```

Comment:
```suggestion
            } else if (n.inputNode.getProperties().get("type").equals("bottom") &&
                       n.preds.size() > 0 &&
```
For consistent placement of `&&` (already a problem before this changeset, but might as well fix now)